code.js

```diff
@@ -0,0 +1,171 @@
+const enemyes = document.querySelectorAll('.enemy');
+const blood = document.querySelector('.blood');
+
+let lastWindow;
+let gameDuration = 30000;
+let enemyKilled = false;
+let score = 0;
+let timeIsOwer = true;
+let gameIsStarted = false;
+let myTimeout;
+let level = 1;
+let enemyName;
+//Listeners
+newGame.addEventListener('click', startGame);
+game.addEventListener('mousedown', (e) => {
+    checkShoot(e);
+});
+settingsButton.addEventListener('click', (e) => {
+    settingsBoard.style.top = e.clientY + 'px';
+    settingsBoard.style.left = e.clientX - 200 + 'px';
+});
+saveSettingsButton.addEventListener('click', saveSettings);
+resetHighScoreButton.addEventListener('click', resetHighScore);
+
+//Functions
+function getRandomTime(min = 600, max = 1500) {
+    return Math.round(Math.random() * (max - min) + min) - 100 * level;
+}
+
+function getRandomWindow() {
+    const rnd = Math.floor(Math.random() * windows.length);
+    const randomWindow = windows[rnd];
+    if (randomWindow === lastWindow) {
+        return getRandomWindow();
+    }
+    lastWindow = randomWindow;
+    return randomWindow;
+}
+
+function showEnemy() {
+    const time = getRandomTime();
+    const window = getRandomWindow();
+    enemyKilled = false;
+    window.classList.add('up');
+    setTimeout(() => {
+        window.classList.remove('up');
+        if (timeIsOwer === false) {
+            showEnemy();
+        }
+    }, time);
+}
+
+function hideAllEnemyes() {
+    windows.forEach((window) => {
+        window.classList.remove('up');
+    });
+}
+
+function checkShoot(e) {
+    if (!e.isTrusted) return;
+    if (e.target.classList.contains('enemy') && enemyKilled === false) {
+        score++;
+        scoreBoard.textContent = `Current score: ${score}`;
+        showBlood(e);
+        enemyKilled = true;
+        if (score % 5 === 0) {
+            levelUp();
+        }
+    }
+}
+
+function showBlood(e) {
+    blood.style.top = e.clientY - 35 + 'px';
+    blood.style.left = e.clientX - 35 + 'px';
+    setTimeout(() => {
+        blood.style.top = '-100px';
+        blood.style.left = '-100px';
+    }, 400);
+}
+
+function setHighScore() {
+    if (!localStorage.highscore) {
+        localStorage.setItem('highscore', score);
+    } else if (localStorage.highscore < score) {
+        localStorage.highscore = score;
+    }
+}
+
+function showHighScore() {
+    let msg = `Highscore: ${localStorage.highscore}`;
+    highScoreBoard.textContent = msg;
+}
+
+function resetHighScore() {
+    localStorage.clear();
+    score = 0;
+    highScoreBoard.textContent = 'Highscore: 0';
+}
+
+function levelUp() {
+    level++;
+    let msg = `Level: ${level}`;
+    levelBoard.textContent = msg;
+}
+
+function saveSettings() {
+    if(Number.isInteger(+setDuration.value) && setDuration.value <= 300) {
+      gameDuration = setDuration.value * 1000;  
+    } else {
+        alert(`Game duration value must be from 0 to 300`);
+    return;
```

Comment:
if you check number firstly, you will can remove unnecessary "return"
something like this
```
    if (isNaN(setDuration.value) || setDuration.value >= 300) {
        alert(`Game duration value must be from 0 to 300`);
    } else {
        gameDuration = setDuration.value * 1000;
        enemyName = setEnemy.value;
        saveEnemy(enemyName);
        hideSettingsPanel();
    }
```